superset/cli.py

```diff
@@ -31,11 +32,47 @@ def init():
     security.sync_role_definitions()
 
 
+def debug_run(app, port, use_reloader):
+    return app.run(
+        host='0.0.0.0',
+        port=int(port),
+        threaded=True,
+        debug=True,
+        use_reloader=use_reloader)
+
+
+def console_log_run(app, port, use_reloader):
+    from console_log import ConsoleLog
```

Comment:
Can you put these imports at the top

Reply:
I know imports should be top-level, but these are optional requirements (see the modified `setup.py`). If they were in the top-level the CLI we would force everybody to install `console_log` (and its dependencies: `gevent`, `gevent-websocket`, etc.).